include/wlr/types/wlr_tablet_tool.h

```diff
@@ -5,8 +5,48 @@
 #include <wayland-server.h>
 #include <wlr/types/wlr_input_device.h>
 
+/*
+ * Copy+Paste from libinput, but this should neither use libinput, nor
+ * tablet-unstable-v2 headers, so we can't include them
+ */
+enum wlr_tablet_tool_type {
+	WLR_TABLET_TOOL_TYPE_PEN = 1,	/**< A generic pen */
+	WLR_TABLET_TOOL_TYPE_ERASER,	/**< Eraser */
+	WLR_TABLET_TOOL_TYPE_BRUSH,	/**< A paintbrush-like tool */
+	WLR_TABLET_TOOL_TYPE_PENCIL,	/**< Physical drawing tool, e.g.
+				             Wacom Inking Pen */
+	WLR_TABLET_TOOL_TYPE_AIRBRUSH,	/**< An airbrush-like tool */
+	WLR_TABLET_TOOL_TYPE_MOUSE,	/**< A mouse bound to the tablet */
+	WLR_TABLET_TOOL_TYPE_LENS,	/**< A mouse tool with a lens */
+};
+
+struct wlr_tablet_tool_tool {
```

Comment:
I'm still not liking this name

Reply:
Neither do I. I'd still prefer a rename to the way tablet-v2 handles it, or good suggestions in general :)